tests/commands/test_commands.py

```diff
@@ -409,3 +409,44 @@ def mock_write(tx, mode):
     resp = run_upsert_validator_new(args, b)
 
     assert b.get_transaction(resp)
+
+
+@pytest.mark.tendermint
+@pytest.mark.bdb
+def test_upsert_validator_show(b, priv_validator_path, user_sk, monkeypatch):
+    from bigchaindb.commands.bigchaindb import run_upsert_validator_show, run_upsert_validator_new
+
+    def mock_get():
+        return [
+            {'pub_key': {'value': 'zL/DasvKulXZzhSNFwx4cLRXKkSM9GPK7Y0nZ4FEylM=',
+                         'type': 'tendermint/PubKeyEd25519'},
+             'voting_power': 10}
+        ]
+
+    def mock_write(tx, mode):
```

Comment:
What is the usage of this mock? I don't think there are any writes.

Reply:
Honestly, it feels like the simplest way to get the data structures I need in place... What with creating the election, and the votes etc, it seems like a bunch of mocking to end up with a data structure that I can get by just using the real code.

Reply:
Refactored the test to use the election fixtures. It no longer uses `run_upsert_validator_new`